### PR TITLE
feat: send emails to team that owns client used by inactive user

### PR DESCRIPTION
**Pull request by NithinKuruba**
Fixes [SSOTEAM-702](https://bcdevex.atlassian.net/browse/SSOTEAM-702)

### CODE REVIEW

File: lambda/app/src/controllers/user.ts [hunk 1]
```diff
@@ -142,19 +145,50 @@ export const isAllowedToManageRoles = async (session: Session, integrationId: nu
...
+            if (u.idir_email === user.email && u.role === 'admin') {
+              isTeamAdmin = true;
+            }
+          });
+          sendTemplate(EMAILS.DELETE_INACTIVE_IDIR_USER, {
+            teamId: integration.teamId,
+            username: user.attributes.idir_username || user.username,
+            clientId: cl.client,
+            roles: cl.roles,
+            teamAdmin: isTeamAdmin,
```
Comment (thegentlemanphysicist) [inline lambda/app/src/controllers/user.ts:172]: so this sends these emails to everyone, not just the  admin?

File: lambda/app/src/routes.ts [hunk 2]
```diff
@@ -128,17 +128,16 @@ export const setRoutes = (app: any) => {
     }
   });
 
-  app.delete(`/users/:userId`, async (req, res) => {
+  app.post(`/delete-inactive-idir-users`, async (req, res) => {
```
Comment (thegentlemanphysicist) [inline lambda/app/src/routes.ts:131]: Why get rid of the delete and turn it into a post request?

File: lambda/app/src/controllers/user.ts (hunk 1 above)
Comment (NithinKuruba) [reply lambda/app/src/controllers/user.ts:172]: yes to all the team members

File: lambda/app/src/routes.ts (hunk 2 above)
Comment (NithinKuruba) [reply lambda/app/src/routes.ts:131]: I can send payload to delete so using post, also updated endpoint name to make it more meaningful

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**